README.md

```diff
@@ -1 +1,271 @@
-# JavaScriptCapstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Most Visitied Countries\] ](#-most-visitied-countries-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Configuring WebPack](#configuring-webpack)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Most Visitied Countries] <a name="about-project"></a>
+
+**[Most Visitied Countries]** is a project that uses the API for its data resource.
```

Comment:
I think you can add more information to the description. For example:

_This project is a web application that provides information about different countries. It allows users to explore details such as the continent, capital, area, and population of various countries. Users can also view comments and add their own insights about specific countries. The application utilizes the Involvement API to enable features like liking countries and commenting. It offers a user-friendly interface with pop-up windows to display country information and interact with the community through comments. The project aims to provide an engaging platform for users to learn and share knowledge about different countries._

Reply:
Hi @sergio-usma. Thanks a lot for your feedback. I'll add the descriptions. It is really great.